recipes/podofo/all/conandata.yml

```diff
@@ -8,7 +8,5 @@ sources:
 patches:
   "0.9.7":
     - patch_file: "patches/0001-fix-cmake-0.9.7.patch"
-      base_path: "source_subfolder"
```

Comment:
Could you please update the conandata with `patch_description` and `patch_type` fields, as described here: https://github.com/conan-io/conan-center-index/blob/master/docs/conandata_yml_format.md#patches-fields. The patch_type can be `conan`.

Reply:
Ok, done